apps/web/src/components/BasicInformation/LanguageInfo.tsx

```diff
@@ -0,0 +1,232 @@
+
+  return (
+    <div
+      data-h2-display="base(grid)"
+      data-h2-grid-template-columns="p-tablet(1fr)"
+      data-h2-gap="base(x1)"
+    >
+      <FieldDisplay label={labels.consideredPositionLanguages}>
+        {lookingForEnglish || lookingForFrench || lookingForBilingual ? (
+          <NoList>
+            <li>
+              <BoolCheckIcon
+                value={lookingForEnglish}
+                trueLabel={intl.formatMessage(commonMessages.interested)}
+                falseLabel={intl.formatMessage(commonMessages.notInterested)}
+              >
+                {intl.formatMessage({
+                  defaultMessage: "English-only positions",
+                  id: "oHIWzo",
+                  description: "Label for language for english field",
+                })}
+              </BoolCheckIcon>
+            </li>
+            <li>
+              <BoolCheckIcon
+                value={lookingForFrench}
+                trueLabel={intl.formatMessage(commonMessages.interested)}
+                falseLabel={intl.formatMessage(commonMessages.notInterested)}
+              >
+                {intl.formatMessage({
+                  defaultMessage: "French-only positions",
+                  id: "BsNhUK",
+                  description: "Label for language for french field",
+                })}
+              </BoolCheckIcon>
+            </li>
+            <li>
+              <BoolCheckIcon
+                value={lookingForBilingual}
+                trueLabel={intl.formatMessage(commonMessages.interested)}
+                falseLabel={intl.formatMessage(commonMessages.notInterested)}
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Bilingual positions",
+                  id: "Mwq0NB",
+                  description: "Label for language for bilingual field",
+                })}
+              </BoolCheckIcon>
+            </li>
+          </NoList>
+        ) : (
+          notProvided
+        )}
+      </FieldDisplay>
+      {lookingForBilingual && (
+        <>
+          <FieldDisplay label={labels.firstOfficialLang}>
+            {firstOfficialLanguage?.label.localized ?? notProvided}
+          </FieldDisplay>
+          <FieldDisplay label={labels.estimatedLanguageAbility}>
+            {estimatedLanguageAbility?.label.localized ? (
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+              >
+                <span>{estimatedLanguageAbility?.label.localized}</span>
+                <span
+                  data-h2-font-weight="base(400)"
+                  data-h2-font-size="base(caption)"
+                  data-h2-color="base(black.light)"
+                >
+                  {intl.formatMessage(
+                    getEstimatedLanguageAbility(estimatedLanguageAbility.value),
+                  )}
+                </span>
+              </div>
```

Comment:
I was checking to make sure all the French loaded, but one thing did not
Not sure why, but this doesn't seem to be working right

![image](https://github.com/user-attachments/assets/1ae3e6a0-9a01-44ce-af01-f6c42b2e1f70)

Reply:
Nice catch. [Fix translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/262835d2d7caf51ee29c8438147a7f777be4a23d).